catalyst-cooperative/pudl
# Merge dev into main for 2023-08-22

- WIP rewrite extractor for backwards compatibility with FTP
- Update EPACEMS extraction to use API only
- First attempt at CEMS 2021-2022 integration, validation needed
- Update doi and etl fast yml, fix unit test
- Fix integration test to reflect changed year data expectations
- Add handling for missing state-year and re-add AK + PR to processing
- Fix plant with missing timezone
- Add missing plants, test missing partitions
- Update release notes
- Fix typo in docs
- Include 2022 in EIA data coverage in README.
- Update dask requirement from <2023.8.1,>=2021.8 to >=2021.8,<2023.8.2
- Update tox requirement from <4.7,>=4 to >=4,<4.10
- Update sphinx requirement from !=5.1.0,<7.2,>=4 to >=4,!=5.1.0,<7.3
- Make sure that epacems caplog actually has records before looping.
- Remove caplog test.
